Packs/AbuseDB/Scripts/AbuseIPDBPopulateIndicators/AbuseIPDBPopulateIndicators.py

```diff
@@ -2,25 +2,64 @@
 from CommonServerPython import *  # noqa: F401
 
 BLACKLISTED = "Bad"
-res = demisto.executeCommand("abuseipdb-get-blacklist", {
-    "days": demisto.args().get("days"),
-    "limit": demisto.args().get("limit"),
-    "confidence": demisto.args().get("confidence")
-})
-
-ips = res[0]['Contents']
-
-if not ips or "Too many requests" in ips:
-    return_error("No Indicators were created (possibly bad API key)")
-
-# Extract IPs into new Indicators
-for ip in ips:
-    demisto.executeCommand("createNewIndicator", {
-        "type": 'ip',
-        "value": ip,
-        "source": 'AbuseIPDB',
-        "reputation": BLACKLISTED,
-        "seenNow": "true",
-    })
-
-demisto.results("All Indicators were created successfully")
+
+
+def get_contents(args: Dict[str, Any]):
+    """Get IP addresses on block lists from AbuseIPDB
+
+    :return: dictionary containing block lists
+    :rtype: ``dict[str, Any]``
+    """
+
+    res = execute_command(
+        "abuseipdb-get-blacklist",
+        {
+            "days": args.get("days"),
+            "limit": args.get("limit"),
+            "confidence": args.get("confidence"),
+        },
+    )
+
+    if not res:
+        return None
+
+    return res[0].get("Contents", None)
+
+
+def check_ips(ips: dict):
+    """Check 'ips' dictionary validity
```

Comment:
Is it a dictionary? I think it is a list (can be checked in the raw response of `"abuseipdb-get-blacklist"`).